src/utils.c

```diff
@@ -1798,7 +1798,11 @@ bool
 as_key_move(as_key* dst, as_key* src)
 {
 	*dst = *src;
-	if (as_load_uint32(&src->value.integer._.count) > 1) {
+	if (!src->valuep) {
+		return true;
+	}
+	if (as_load_uint32(&src->valuep->integer._.count) > 1) {
```

Comment:
nit: we usually put an empty line between if statements. Unless there is something else specified in Clayton's style guide.